src/pages/Home.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export default function Home() {
+  return (
+    <div>
+      <h4>Welcome to our page !</h4>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+      <p> Curabitur pretium convallis tortor id venenatis.</p>
+      <p> Aenean quis justo ut orci aliquam mollis. Integer condimentum a </p>
+      <p> volutpat eget tellus ut, mollis ornare tortor</p>
+    </div>
+  );
+}
```

Comment:
- [ ] You can make your home page looks better by providing some styling on the project like Background color and making more content. You can look for the general layout for reference
### your design
![his](https://user-images.githubusercontent.com/81727988/180721165-4fd2e7e4-77bb-4fe7-90eb-38d41cbb7d61.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/180721216-cae37405-aa0d-405c-a311-1cea17cd263f.png)